### PR TITLE
Add functional tests - updating performance profile

### PR DESCRIPTION
**Pull request by dshchedr**
In this func test we are verifying that all profile parameters can be changed. Unfortunately, after running it - some other functional tests might be failed. 
 I didn't find good solution how to postpone running this test until all other will be finished, so I would be much appreciated for any advises - what is the best way to manage it.

### CODE REVIEW

File: functests/performance/performance.go [hunk 1]
```diff
@@ -170,8 +173,60 @@ var _ = Describe("performance", func() {
...
+			profile, err = profiles.GetByNodeLabels(
+				testclient.Client,
+				map[string]string{
+					fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerRT): "",
+				},
+			)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("Verifies that hugepages can be updated thru performance profile", func() {
```
Comment (gsr-shanks) [inline functests/performance/performance.go]: Let's add the test case id's and other parameters while we are at it.

File: functests/performance/performance.go [hunk 2]
```diff
@@ -170,8 +173,60 @@ var _ = Describe("performance", func() {
 			execSysctlOnWorkers(workerRTNodes, sysctlMap)
 		})
 	})
+
+	Describe("Updating parameters in performance profile", func() {
```
Comment (gsr-shanks) [inline functests/performance/performance.go]: Let's add the rfe id while we are at it.

File: functests/performance/performance.go (hunk 1 above)
Comment (dshchedr) [reply functests/performance/performance.go]: Will do, but at the end this test should combine several polarion tests.. do you know how it's better to mark all of them?

**Comment by slintes** [comment]
/ok-to-test

@dshchedr will review on monday! :)

File: functests/performance/performance.go (hunk 2 above)
Comment (dshchedr) [reply functests/performance/performance.go]: added [rfe_id:28761]

File: functests/performance/updating_profile.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+					Size:  hpSize,
+				},
+			},
+		}
+		profile.Spec.CPU = &performancev1alpha1.CPU{
+			BalanceIsolated: &f,
+			Reserved:        &reserved,
+			Isolated:        &isolated,
+		}
+		profile.Spec.NUMA.TopologyPolicy = &policy
```
[the remark this answers is not in the document]
Comment (dshchedr) [reply functests/performance/updating_profile.go]: done

File: functests/performance/updating_profile.go [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+		profile.Spec.RealTimeKernel.Enabled = &f
```
[the remark this answers is not in the document]
Comment (dshchedr) [reply functests/performance/updating_profile.go]: done

**Pull request merged**